AppCUI/src/Application/Application.cpp

```diff
@@ -930,6 +952,21 @@ bool AppCUI::Internal::Application::ExecuteEventLoop(Control* ctrl)
 
     while (LoopStatus == LOOP_STATUS_NORMAL)
     {
+        if (!toDelete.empty())
```

Comment:
is this required? callul la clear() impacteaza in vreun fel pe un array empty?

Reply:
e o bucla de evenimente - preferam sa facem ceva cat mai repede posibil (empty e preferat de verific pentru ca e cel mai rapid - e si inline, si const, etc).

Reply:
ok